ovos_utils/skills/locations.py

```diff
@@ -0,0 +1,123 @@
+from os.path import join, isdir, dirname, expanduser, isfile
+from os import makedirs, listdir
+from typing import List, Optional
+from ovos_config.locations import get_xdg_data_save_path, get_xdg_data_dirs
+from ovos_config.config import Configuration
+from ovos_utils.log import LOG
+
+
+def get_installed_skill_ids(config=None) -> List[str]:
+    """
+    Gets a list of `skill_id`s for all installed skills
+    @param config:
+    @return:
+    """
+    skill_ids = get_plugin_skill_dirs()
```

Comment:
is this correct? should not return paths but rather skill_id

skill_id corresponds to base folder name in non pip installs, otherwise the plugin provides the skill_id directly

Reply:
Right, these are directories; I'll define a separate method for getting skill dirs and see if I can work with OPM to get skill_ids here